homeassistant/components/hausbus/device.py

```diff
@@ -0,0 +1,158 @@
+                    case 0x19:
+                        self.model_id = "4-fach Taster"
+                    case 0x1A:
+                        self.model_id = "2-fach Taster"
+                    case 0x1B:
+                        self.model_id = "1-fach Taster"
+                    case 0x1C:
+                        self.model_id = "6-fach Taster Gira"
+                    case 0x20:
+                        self.model_id = "32-fach IO"
+                    case 0x0C:
+                        self.model_id = "16-fach Relais"
+                    case 0x08:
+                        self.model_id = "8-fach Relais"
+                    case 0x10:
+                        self.model_id = "22-fach UP-IO"
+                    case 0x28:
+                        self.model_id = "8-fach Dimmer"
+                    case 0x30:
+                        self.model_id = "2-fach RGB Dimmer"
+                    case 0x00:
+                        self.model_id = "4-fach 0-10V Dimmer"
+                    case 0x01:
+                        self.model_id = "4-fach 1-10V Dimmer"
+                    case _:
+                        self.model_id = "Controller"
+            case EFirmwareId.HBC:
+                match type_id:
+                    case 0x18:
+                        self.model_id = "6-fach Taster"
+                    case 0x19:
+                        self.model_id = "4-fach Taster"
+                    case 0x1A:
+                        self.model_id = "2-fach Taster"
+                    case 0x1B:
+                        self.model_id = "1-fach Taster"
+                    case 0x1C:
+                        self.model_id = "6-fach Taster Gira"
+                    case 0x20:
+                        self.model_id = "32-fach IO"
+                    case 0x0C:
+                        self.model_id = "16-fach Relais"
+                    case 0x08:
+                        self.model_id = "8-fach Relais"
+                    case 0x10:
+                        self.model_id = "24-fach UP-IO"
+                    case 0x28:
+                        self.model_id = "8-fach Dimmer"
+                    case 0x29:
+                        self.model_id = "8-fach Dimmer"
+                    case 0x30:
+                        self.model_id = "2-fach RGB Dimmer"
+                    case 0x00:
+                        self.model_id = "4-fach 0-10V Dimmer"
+                    case 0x01:
+                        self.model_id = "4-fach 1-10V Dimmer"
+                    case _:
+                        self.model_id = "Controller"
+            case EFirmwareId.SD485:
+                match type_id:
+                    case 0x28:
+                        self.model_id = "24-fach UP-IO"
+                    case 0x1E:
+                        self.model_id = "6-fach Taster"
+                    case 0x2E:
+                        self.model_id = "6-fach Taster"
+                    case 0x2F:
```

Comment:
duplicate code in here. Keep it in english please.

Reply:
These are the official product names, but I will try to translate.

Reply:
ideally, you'd just set translation_keys and add the (english) names to `strings.json` so they can get translated automatically https://developers.home-assistant.io/docs/internationalization/core#devices